docs/Using-Fleet/FAQ.md

```diff
@@ -318,3 +334,13 @@ Run `sudo apt remove fleet-osquery -y`
 
 ### CentOS
 Run `sudo rpm -e fleet-osquery-X.Y.Z.x86_64`
+
+## How does Fleet determines online and offline status?
+
+- *Online* hosts are devices that are connected to Fleet and will respond to a live query.
+Fleet considers a host as online if it has connected successfully in the last `distributed_interval` (or `config_tls_refresh`, whichever is smaller) seconds.
```

Comment:
Is "seconds" supposed to be at the end of this sentence? It seems confusing since it doesn't specify how many seconds.

Reply:
Yes, `distributed_interval` is a value e.g. `60` and represents seconds.

Let me know if there's a better way to make this clear.

Reply:
It looks like you explain that it's 60 seconds in the following sentence, so we could just cut the instance of "seconds" here.

Reply:
I'll use two examples, let me know if it matches what's explained:

`distributed_interval=10`,`config_tls_refresh=30`
A host is considered online if it has connected to Fleet in the last 70 (10+60) seconds.

`distributed_interval=30`,`config_tls_refresh=20`
Then a host is considered online if it has connected to Fleet in the last 80 (20+60) seconds.

(The +60 seconds is a hard-set buffer.)

Reply:
That certainly provides more detail. The phrase "hard-set buffer" is confusing me a little . I'd use "set buffer" instead. But if "hard-set buffer" is commonly used, feel free to leave it as is.

Reply:
By hard-set I mean it's a fixed value of 60 seconds (we also use the term "hardcoded", cannot be changed).

Reply:
> That certainly provides more detail. 

OK, feel free to make edits or propose alternative ways of explaining this.

Reply:
I've pushed another try. Let me know if that looks good.